src/pywrapper.cpp

```diff
@@ -102,26 +116,61 @@ bool PyWrapper::init()
     // communication channel for I/O Intr value exchange
     PyImport_AppendInittab("pydev", &PyInit_pydev);
 
+    // Py_SetProgramName(L"python3.7");
+    std::cerr << "Initalising python3" << std::endl;
     Py_Initialize();
+    std::cerr << "Initialsing threads" << std::endl;
     PyEval_InitThreads();
 
+    std::cerr << "Initialsing dics" << std::endl;
+
+#if PY_MAJOR_VERSION == 3
+
+    /*
+     * IN Python 3 the
+     */
+    auto m = PyImport_AddModule("__main__");
+    assert(m);
+    globDict = PyModule_GetDict(m);
+    assert(globDict);
+    locDict = PyDict_New();
+    assert(locDict);
+    /*     */
+#elif PY_MAJOR_VERSION == 2
     globDict = PyDict_New();
     locDict = PyDict_New();
     PyDict_SetItemString(globDict, "__builtins__", PyEval_GetBuiltins());
+#else /* PY_MAJOR_VERSION */
+#error "Unsupported python version"
+#endif /* PY_MAJOR_VERSION */
 
     mainThread = PyEval_SaveThread();
 
     // Make `pydev' module appear as built-in module
+    std::cerr << "Importing pydev" << std::endl;
```

Comment:
Let's not do stdout or stderr logging from library.

Reply:
This line is of course a left over one. 

What' do you think on sending the "offending command" when PyERR_PRint is used (e.g. https://github.com/hz-b/PyDevice/blob/73f8ad9e9389beefbb1f90cdc8e7b7cfe0b3dcab/src/pywrapper.cpp#L393)? I thought that would emit its messages to stdout or stderr anway.